core/container.go

```diff
@@ -553,7 +550,7 @@ func (container *Container) ExitCode(ctx context.Context, gw bkgw.Client) (*int,
 		return nil, err
 	}
 
-	if file != nil {
+	if file == nil {
```

Comment:
if the file is `nil` doesn't it mean we should return some error?

Reply:
This is part of the "happy path" from an API perspective: if the file doesn't exist, it just means the command hasn't run yet. The API returns a null `exitCode` in that case.